inst/css/filter-panel.css

```diff
@@ -267,11 +279,16 @@ a.remove_all:hover {
   border-bottom-right-radius: 0;
 }
 
-.filter-card-header:hover,
-.filter-card-body:hover {
+.filter-card-header:hover {
```

Comment:
```suggestion
.filter-card-header:hover {
  cursor: pointer;
```
One important requirement to show that the filter-header card can be clicked was to implement a cursor pointer.

Reply:
In that case I would recommend a mild scale transition along with increased opacity on the close buttons on hover WDYT?

Reply:
We already seem to do the opacity change to the outer buttons
<img width="415" alt="Screenshot 2024-03-11 at 6 29 22 PM" src="https://github.com/insightsengineering/teal.slice/assets/49812166/bd909253-7291-4423-929c-89e26abd06be">

Reply:
When you change filter state, the rewind and reset buttons will appear in the header, which have full opacity.

Reply:
I tried the pointer suggestion and I think this can easily be missed by user.